.travis.yml

```diff
@@ -33,9 +34,8 @@ matrix:
         homebrew:
           update: true
           packages:
-            - boost@1.70
             - ccache
-script: "./.cicd/cicd-build.sh"
+script: "./.cicd/run.sh"
```

Comment:
Given the complexity of this BASH, I think we need to go back to separate build and test scripts.